### PR TITLE
Resolving Design Requests in Community page

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Notes for Reviewers**
- Removed unused Variables
- Incorporated design update requests
  - Early access section description update 
  - Handbook aspect ratio fix
  - Inappropriate capitalization fix 
  - Fixed Responsive inconsistencies
- Additionally Updated hero section  

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
-  [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@thisiskaransgit Let's discuss it on the websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

File: _includes/modal.html [hunk 1]
```diff
@@ -27,7 +27,7 @@ <h4 class="related-patterns" style="text-transform: uppercase; margin-top: 2rem;
       <div>
         {%if pattern.patternInfo%}
         <h3 class="modal-h3">What this pattern does</h3>
-        <p>{{pattern.patternInfo}}</p>
+        <p>{{pattern.patternInfo | replace: '\\n', '<br/>'| replace: '\n', '<br/>' }}</p>
```
Comment (thisiskaransgit) [inline _includes/modal.html:30]:
```suggestion
        <p>{{pattern.patternInfo}}</p>
```

File: _includes/partials/individual-component.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h4 class="related-patterns">Related Patterns</h4>
   {%if pattern.patternInfo%}
   <div>
     <h3>What this pattern does:</h3>
-    <p>{{pattern.patternInfo}}</p>
+    <p>{{pattern.patternInfo | replace: '\\n', '<br/>'| replace: '\n', '<br/>' }}</p>
```
Comment (thisiskaransgit) [inline _includes/partials/individual-component.html:17]:
```suggestion
    <p>{{pattern.patternInfo}}</p>
```

File: collections/_pages/community.html [hunk 3]
```diff
@@ -2,109 +2,93 @@
...
     <div class="text-container">
-      <p class="heading">User? Contributor? No matter.</p>
-      <p class="title">PARTICIPATE</p>
-      <a href="https://slack.meshery.io/" class="banner-button">
-        <p class="banner-button-text">Welcome Aboard</p>
-      </a>
+      <p class="heading">Empower Yourself</p>
+      <p class="community-heading">Engage, Contribute, and Make a Difference</p>
     </div>
+    <a href="https://slack.meshery.io/" class="highlight">Welcome Aboard</a>
```
Comment (Chadha93) [inline collections/_pages/community.html:16]:
```suggestion
    <a href="https://slack.meshery.io/" class="highlight">Join Our Talented Community</a>

```

File: collections/_pages/community.html [hunk 4]
```diff
@@ -2,109 +2,93 @@
...
 <section class="banner-section">
-  <div class="hero">
+  <div class="community-hero">
     <div class="text-container">
-      <p class="heading">User? Contributor? No matter.</p>
-      <p class="title">PARTICIPATE</p>
-      <a href="https://slack.meshery.io/" class="banner-button">
-        <p class="banner-button-text">Welcome Aboard</p>
-      </a>
+      <p class="heading">Empower Yourself</p>
```
Comment (Chadha93) [inline collections/_pages/community.html:13]:
```suggestion
      <p class="heading">The Meshery Community!</p>

```

Comment (Chadha93) [reply collections/_pages/community.html:13]: Make it H1

File: collections/_pages/community.html [hunk 5]
```diff
@@ -2,109 +2,93 @@
...
+      <p class="community-heading">Engage, Contribute, and Make a Difference</p>
```
[the remark this answers is not in the document]
Comment (Chadha93) [reply collections/_pages/community.html:14]: Make this sub-heading of H1

File: collections/_pages/community.html [hunk 6]
```diff
@@ -2,109 +2,93 @@
...
     </div>
+    <a href="https://slack.meshery.io/" class="highlight">Welcome Aboard</a>
   </div>
 </section>
 
 <!-- Resources Section -->
 <section class="sectionContainer flex">
   <h1 class="head">Resources</h1>
-  <p class="paragraph">More than just a community of contributors, we are also a support System</p>
+  <p class="paragraph">More than just a community of contributors, we are also a support system</p>
```
Comment (Chadha93) [inline collections/_pages/community.html:23]:
```suggestion
  <p class="paragraph">Warm, welcoming, and encouraging. Embrace developer-defined infrastructure and help empower every engineer with us.</p>

```

File: collections/_pages/community.html [hunk 7]
```diff
@@ -2,109 +2,93 @@
...
   <p class="paragraph para2">We Know you'll need all the help you can get, and we provide it through.</p>
```
Comment (Chadha93) [inline collections/_pages/community.html:24]:
```suggestion
```

File: collections/_pages/community.html [hunk 8]
```diff
@@ -119,12 +103,7 @@ <h2 class="margin-padding-none">Early Access</h2>
     <div class="text flex-item">
       <h4 class="handbook-header">EXPLORE THE LAYER5 COMMUNITY</h4>
```
Comment (Chadha93) [inline collections/_pages/community.html:104]:
```suggestion
      <h4 class="handbook-header">Jump in. The community is warm and welcoming!</h4>

```

File: collections/_pages/community.html [hunk 9]
```diff
@@ -119,12 +103,7 @@ <h2 class="margin-padding-none">Early Access</h2>
     <div class="text flex-item">
       <h4 class="handbook-header">EXPLORE THE LAYER5 COMMUNITY</h4>
       <h1 class="sub-header">Community Handbook</h1>
-      <p class="handbook-paragraph">
-        The community is growing at a tremendous rate. We value our connections
-        and that is what makes us unique. Whether it is your first meeting or
-        your tenth contribution, we want to make sure that you are supported and
-        equipped to take advantage of all of that the community has to offer.
-      </p>
+      <p class="handbook-paragraph">The community is growing at a tremendous rate. We value our connections and that is what makes us unique. Whether it is your first meeting or your tenth contribution, we want to make sure that you are supported and equipped to take advantage of all of that the community has to offer.</p>
```
Comment (Chadha93) [inline collections/_pages/community.html:106]:
```suggestion
      <p class="handbook-paragraph">With active members from around the globe, the community is a collection of diverse open source contributors, each of whom espouse our cultural values of inclusivity and paying it forward for others. We welcome individuals at all stages of their development and of all walks of life. Every member is encourage to embrace these values and support one another in meaningfully and consistently advancing the community's open source initiatives.</p>

```

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@thisiskaransgit The hero section needs work on the graphic (cc @Rexford74 @ritiksaxena124), let's use some of the white space.

The button seems inconsistent.

**Pull request closed without merging**